utils/threshAlgos.py

```diff
@@ -451,7 +451,9 @@ def calibrateThrDACStar(inputs):
     """
     return calibrateThrDAC(*inputs)
 
-def calibrateThrDAC(inputFile,fitRange="0,255",listOfVFATs=None,noLeg=False,outputDir=None,savePlots=False,debug=False):
+from anaInfo import maxEffPedPercentDefault, highNoiseCutDefault, deadChanCutLowDefault, deadChanCutHighDefault
+
+def calibrateThrDAC(inputFile,fitRange="0,255",listOfVFATs=None,maxEffPedPercent=maxEffPedPercentDefault,highNoiseCut=highNoiseCutDefault,deadChanCutLow=deadChanCutLowDefault,deadChanCutHigh=deadChanCutHighDefault,noLeg=False,outputDir=None,savePlots=False,debug=False):
```

Comment:
consider passing a namespace rather than a large argument list and then checking if expected entries exist and and if they do not exist either add them with a default option or raise an appropriate exception.

Reply:
done